yt_dlp/extractor/locipo.py

```diff
@@ -0,0 +1,149 @@
+        return self._download_json(
+            f'https://api.locipo.jp/api/v1/{path}', item_id)
+
+
+class LocipoIE(LocipoBaseIE):
+    _VALID_URL = r'https?://locipo\.jp/(?P<type>creative|embed)/(?P<id>[^/?#]+)?'
+
+    _TESTS = [{
+        'url': 'https://locipo.jp/creative/fb5ffeaa-398d-45ce-bb49-0e221b5f94f1',
+        'info_dict': {
+            'id': 'fb5ffeaa-398d-45ce-bb49-0e221b5f94f1',
+            'ext': 'mp4',
+            'title': 'リアルカレカノ#4 ～伊達さゆりと勉強しよっ？～',
+            'description': 'md5:70a40c202f3fb7946b61e55fa015094c',
+            'duration': 3622,
+            'genres': 'count:1',
+            'modified_date': '20240904',
+            'modified_timestamp': 1725415481,
+            'release_timestamp': 1711789200,
+            'release_date': '20240330',
+            'series': 'リアルカレカノ',
+            'series_id': 'b865b972-99fe-41d5-a72c-8ed5c42132bd',
+            'thumbnail': r're:https?://.+\.(?:jpg|png)',
+            'timestamp': 1711789200,
+            'upload_date': '20240330',
+            'uploader': '東海テレビ',
+            'uploader_id': 'thk',
+        },
+    }, {
+        'url': 'https://locipo.jp/embed/?id=71a334a0-2b25-406f-9d96-88f341f571c2',
+        'info_dict': {
+            'id': '71a334a0-2b25-406f-9d96-88f341f571c2',
+            'ext': 'mp4',
+            'title': '#1 オーディション／ゲスト伊藤美来、豊田萌絵',
+            'description': 'md5:5bbcf532474700439cf56ceb6a15630e',
+            'duration': 3399,
+            'modified_date': '20250704',
+            'modified_timestamp': 1751623614,
+            'release_timestamp': 1751623200,
+            'release_date': '20250704',
+            'series': '声優ラジオのウラカブリ～Locipo出張所～',
+            'series_id': 'eaf2f2b2-aa73-40f1-b4c9-e47f098775b8',
+            'thumbnail': r're:https?://.+\.(?:jpg|png)',
+            'timestamp': 1751623200,
+            'upload_date': '20250704',
+            'uploader': 'テレビ愛知',
+            'uploader_id': 'tva',
+        },
+    }, {
+        'url': 'https://locipo.jp/creative/860201fa-f22b-4ffd-8890-6320a857159f?list=fef7c4fb-741f-4d6a-a3a6-754f354302a2',
+        'info_dict': {
+            'id': 'fef7c4fb-741f-4d6a-a3a6-754f354302a2',
+            'title': 'CBCアナウンサー公式【みてちょてれび】',
+            'description': 'md5:50a1b23e63112d5c06c882835c8c1fb1',
+            'genres': 'count:1',
+            'modified_date': '20250611',
+            'modified_timestamp': 1749614012,
+            'thumbnail': r're:https?://.+\.(?:jpg|png)',
+        },
+        'playlist_mincount': 32,
+        'params': {'yes_playlist': True},
+    }]
+
+    def _real_extract(self, url):
+        video_type, video_id = self._match_valid_url(url).group('type', 'id')  # type: ignore
+        if not video_id:
+            video_id = traverse_obj(parse_qs(url), ('id', -1, {str}, {require('video ID')}))
+        playlist_id = traverse_obj(parse_qs(url), ('list', -1, {str}))
+        if playlist_id and self._yes_playlist(playlist_id, video_id):
```

Comment:
`--yes-playlist` is the default behavior and can only be disabled with `--no-playlist`, so you were right and I was mistaken, sorry
```suggestion
        if playlist_id and not self.get_param('noplaylist'):
```